risc0/zkvm/src/exec/executor.rs

```diff
@@ -12,9 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! This module implements the Executor.
-
-use std::{cell::RefCell, fmt::Debug, io::Write, mem::take, rc::Rc};
+/// This module implements the Executor.
```

Comment:
Is using `///` better than using `//!` in this context? I was under the impression that `//!` was what we wanted.

Reply:
this was unintentional. Fixed!